addons/web/static/src/core/datetime/datetimepicker_service.js

```diff
@@ -408,6 +416,24 @@ export const datetimePickerService = {
                             }
                         }
                     );
+                }
+
+                /**
+                 * When the datetime input is an operation as +=3d or -=5w for
+                 * incremental mass edit, the mechanisms of picker is aborted.
```

Comment:
I would say incremental edit, not mass edit, as mass edit is a derivation of the incremental edit feature